src/app/game-state/followers.service.ts

```diff
@@ -248,12 +251,13 @@ export class FollowersService {
     this.followersUnlocked = properties.followersUnlocked || false;
     this.autoDismissUnlocked = properties.autoDismissUnlocked || false;
     this.maxFollowerByType = properties.maxFollowerByType || {};
+    this.followerLifespanDoubled = properties.followerLifespanDoubled || false;
   }
 
   generateFollower(){
     this.followersRecruited++;
     if (this.followers.length >= this.followerCap){
-      this.logService.addLogMessage("A new follower shows up, but you already have too many. You are forced to turn them away.","INJURY","FOLLOWER");
+      this.logService.addLogMessage("A new follower shows up, but you already have too many. You are forced to turn them away.","INJURY","EVENT");
```

Comment:
I don't understand why you made this change, wasn't the idea that you should be able to hide these unnecessary logs?
At the stage of the game i am at you get one of these every ~4 seconds.

Reply:
This change is because it's a reaction to player input, since there should be feedback for the activity, especially when a limit is hit or the action succeeds. If someone could add a section for ACTIVITY that would be great as well, for people that don't want activity feedback.

Reply:
So I figured out how to turn it red, and then added a button in settings to make testing simpler and to fill out a player request, though I don't know where that request is anymore. PR is separate.